experimental/dds/tree2/src/test/feature-libraries/editable-tree/editableTree.editing.spec.ts

```diff
@@ -327,9 +301,10 @@ describe("editable-tree: editing", () => {
 
 	describe(`can move nodes`, () => {
 		it("to the left within the same field", () => {
-			const tree = createSharedTree(getTestSchema(FieldKinds.sequence), [
-				{ type: rootSchemaName },
-			]);
+			const tree = viewWithContent({
```

Comment:
Once the separation between the shared object SharedTree and its related view interfaces (which correspond to branches) is sorted out, it might be worth trying to normalize these names.